src/libmongoc/doc/configuring_tls.rst

```diff
@@ -23,7 +23,7 @@ The following URI options may be used to further configure TLS:
 Configuration with mongoc_ssl_opt_t
 -----------------------------------
 
-Alternatively, the :symbol:`mongoc_ssl_opt_t` struct may be used to configure TLS with :symbol:`mongoc_client_set_ssl_opts()` or :symbol:`mongoc_client_pool_set_ssl_opts()`. Most of the configurable options can be using the Connection URI.
+Alternatively, the :symbol:`mongoc_ssl_opt_t` struct may be used to configure TLS with :symbol:`mongoc_client_set_ssl_opts()` or :symbol:`mongoc_client_pool_set_ssl_opts()`. Most of the configurable options can be set using the `Connection String URI <http://docs.mongodb.org/manual/reference/connection-string/>`_.
```

Comment:
Nit, but we should probably use `https` whenever possible.

Reply:
Good point. I'll do a global find-replace for `docs.mongodb.org` links in a follow-up commit. Though at least docs.mongodb.org has redirects in place at least.